exts/betterTooltip/betterTooltip.js

```diff
@@ -47,14 +94,63 @@
                 return cd;
             }
 
+            /**
+             * Installs this extension in a given chart definition.
+             *
+             * The formatter instance itself can be called as a function,
+             * being equivalent to calling this method.
+             *
+             * This function defaults the properties:
+             * * `tooltipEnabled` — `true`
+             * * `tooltipOpacity` — `1`
+             * * `tooltipGravity` — `"s"`
+             * * `tooltipClassName` — `"ccc-ext-better-tooltip"`
+             * * `tooltipFollowMouse` — `true`
+             *
+             * This function sets the required property: `tooltipFormat`.
+             *
+             * @name pvc.ext.BetterTooltip#install
+             * @function
+             * @param {Object} cd The chart definition to extend.
+             * @param {boolean} [defaults=false] Indicates that
+             * only required or optional properties not present in the chart definition are set.
+             * @return {pvc.ext.BetterTooltip} This instance.
+             */
             formatter.install = formatter;
 
+            /**
+             * Formats an HTML tooltip for a given scene.
+             *
+             * This function can be called on any `this` context,
+             * and will always exhibit the same behavior.
+             *
+             * Normally you would not use this function directly,
+             * as {@link pvc.ext.BetterTooltip#install}
+             * sets this as the chart"s `tooltipFormat` for you.
+             *
+             * @alias format
+             * @memberOf pvc.ext.BetterTooltip#
+             * @function
+             * @param {pvc.visual.Scene} scene The categorical scene for which to render the tooltip.
+             * @return {string} The tooltip HTML string.
+             */
             formatter.format = function(scene) {
                 var model = buildModel.call(formatter, scene);
 
                 return betterTooltipRenderer.call(formatter, model);
             };
 
+            /**
+             * Gets or sets the format string for the label of the category. Used by the default label formater.
```

Comment:
Gets or sets the format string for the label of the category. Used by the default label formatter.